### PR TITLE
Added styling and general component HTML of event types ui

### PR DESCRIPTION
**Pull request by friedow**

### CODE REVIEW

**Comment by hpibptlab** [comment]
SonarQube analysis reported 3 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [App-styles.js](https://github.com/bptlab/argos-frontend/blob/e0c080c4098537c12db66c6d66812319307d958c/src/App-styles.js): Rename this file to "appStyles". [rule](http://bpt-lab.org/sonarqube/coding_rules#rule_key=javascript%3AS3317)
1. ![MAJOR][MAJOR] [Definition-styles.js](https://github.com/bptlab/argos-frontend/blob/e0c080c4098537c12db66c6d66812319307d958c/src/Definition-styles.js): Rename this file to "definitionStyles". [rule](http://bpt-lab.org/sonarqube/coding_rules#rule_key=javascript%3AS3317)
1. ![MAJOR][MAJOR] [ErrorMessage-styles.js](https://github.com/bptlab/argos-frontend/blob/e0c080c4098537c12db66c6d66812319307d958c/src/Utils/ErrorMessage-styles.js): Rename this file to "errorMessageStyles". [rule](http://bpt-lab.org/sonarqube/coding_rules#rule_key=javascript%3AS3317)

[MAJOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR'

**Review by tommartensen — APPROVED** [review]

File: src/EventTypesView/EventQueryListItem.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react';
+import IconButton from 'material-ui/IconButton';
+import IconEdit from 'material-ui/svg-icons/editor/mode-edit';
+import IconDelete from 'material-ui/svg-icons/action/delete';
+import {ListItem} from 'material-ui/List';
+
+
+class EventQueryListItem extends Component {
+	deleteEventQuery() {
+		// TODO: to be filled
```
Comment (hpibptlab) [inline src/EventTypesView/EventQueryListItem.js:10]: INFO Complete the task associated to this TODO comment. [rule](http://bpt-lab.org/sonarqube/coding_rules#rule_key=javascript%3AS1135)

File: src/EventTypesView/EventQueryListItem.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+	}
+
+	editEventQuery() {
+		// TODO: to be filled
```
Comment (hpibptlab) [inline src/EventTypesView/EventQueryListItem.js:14]: INFO Complete the task associated to this TODO comment. [rule](http://bpt-lab.org/sonarqube/coding_rules#rule_key=javascript%3AS1135)

**Comment by hpibptlab** [comment]
SonarQube analysis reported 5 issues
* ![MAJOR][MAJOR] 3 major
* ![INFO][INFO] 2 info

Watch the comments in this conversation to review them.

#### 3 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [App-styles.js](https://github.com/bptlab/argos-frontend/blob/8876488468f6a59a281b4aa85add99fa46adc3ed/src/App-styles.js): Rename this file to "appStyles". [rule](http://bpt-lab.org/sonarqube/coding_rules#rule_key=javascript%3AS3317)
1. ![MAJOR][MAJOR] [Definition-styles.js](https://github.com/bptlab/argos-frontend/blob/8876488468f6a59a281b4aa85add99fa46adc3ed/src/Definition-styles.js): Rename this file to "definitionStyles". [rule](http://bpt-lab.org/sonarqube/coding_rules#rule_key=javascript%3AS3317)
1. ![MAJOR][MAJOR] [ErrorMessage-styles.js](https://github.com/bptlab/argos-frontend/blob/8876488468f6a59a281b4aa85add99fa46adc3ed/src/Utils/ErrorMessage-styles.js): Rename this file to "errorMessageStyles". [rule](http://bpt-lab.org/sonarqube/coding_rules#rule_key=javascript%3AS3317)

[INFO]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO'
[MAJOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR'

**Comment by hpibptlab** [comment]
SonarQube analysis reported 2 issues
* ![INFO][INFO] 2 info

Watch the comments in this conversation to review them.

[INFO]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO'

**Pull request merged**